docs/source/extension/extension_tutorial.rst

```diff
@@ -393,7 +418,7 @@ The single *Astronomy Picture* tab should come to the foreground.
 
 If your widget is not behaving, compare your code with the reference
 project state at the `01-show-a-panel
-tag <https://github.com/jupyterlab/jupyterlab_apod/tree/3.0-01-show-a-panel>`__.
+tag <https://github.com/ericsnekbytes/apod_extension/tree/01-show-a-panel>`__.
```

Comment:
Could you restore the old link as we should update the old repo?

Reply:
This is done (new links to the official repo are in).